CallAutomation_SimpleIvr/Program.cs

```diff
@@ -131,15 +134,47 @@
         }
         if (@event is PlayCompleted)
         {
+            //if (@event.OperationContext == "AgentConnect")
+            //{
+            //    var addParticipantOptions = new AddParticipantsOptions(new List<CommunicationIdentifier>()
+            //    {
+            //        new PhoneNumberIdentifier(builder.Configuration["ParticipantToAdd"])
+            //    });
+
+            //    addParticipantOptions.SourceCallerId = new PhoneNumberIdentifier(builder.Configuration["ACSAlternatePhoneNumber"]);
+            //    await callConnection.AddParticipantsAsync(addParticipantOptions);
+            //}
             if (@event.OperationContext == "AgentConnect")
             {
-                var addParticipantOptions = new AddParticipantsOptions(new List<CommunicationIdentifier>()
+                var SourceCallerId = new PhoneNumberIdentifier(builder.Configuration["ACSAlternatePhoneNumber"]);
+                var target = builder.Configuration["ParticipantToAdd"];
+                var Participants = target.Split(';');
+                var count = 0;
+                foreach (var Participantindentity in Participants)
                 {
-                    new PhoneNumberIdentifier(builder.Configuration["ParticipantToAdd"])
-                });
-
-                addParticipantOptions.SourceCallerId = new PhoneNumberIdentifier(builder.Configuration["ACSAlternatePhoneNumber"]);
-                await callConnection.AddParticipantsAsync(addParticipantOptions);
+                    var Target = new PhoneNumberIdentifier(Participantindentity);
+
+                    var CallInvite = new CallInvite(Target, SourceCallerId);
+                    var addParticipantOptions = new AddParticipantOptions(CallInvite);
+                    await callConnection.AddParticipantAsync(addParticipantOptions);
+                    count++;
+                    logger.LogInformation($"Add Participant: {JsonConvert.SerializeObject(@event)}" + $"participant id :{Participantindentity}");
+                }
+                logger.LogInformation($"List of Participants: {count}" + $"� participant ids :{target}");
```

Comment:
remove the ? marks unrecognized char.

Reply:
I am not changing anything on SimpleIVR ,